BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/BrainCloudComms.cs

```diff
@@ -499,10 +499,41 @@ public void Update()
                 }
                 else if (status == RequestState.eWebRequestStatus.STATUS_DONE)
                 {
+#if USE_WEB_REQUEST
+                    if (_activeRequest.WebRequest.responseCode == 200)
+                    {
+                        ResetIdleTimer();
+                        HandleResponseBundle(GetWebRequestResponse(_activeRequest));
+
+                        _activeRequest = null;        
+                    }
+                    else if (_activeRequest.WebRequest.responseCode == 503)
+                    {
+                        //Packet in progress
+                        _clientRef.Log("Packet in progress");
+                        RetryRequest(status, bypassTimeout);
+                        return;
+                    }
+                    else
+                    {
+                        //Error Callback
+                        var errorResponse = GetWebRequestResponse(_activeRequest);
+                        if (_serviceCallsInProgress.Count > 0)
+                        {
+                            ServerCallback sc = _serviceCallsInProgress[0].GetCallback();
+                            if (sc != null)
+                            {
+                                sc.OnErrorCallback(404,(int)_activeRequest.WebRequest.responseCode, errorResponse);    
+                            }
+                        }
+                    }
+#elif DOT_NET
```

Comment:
Why we don't do it for .net also?

Reply:
I attempted to but couldn't find a way to get the response code for .net. I will take another look to see if I can do this for .net also.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpresponsemessage.statuscode?view=net-5.0#System_Net_Http_HttpResponseMessage_StatusCode

Reply:
Cast that into an int